Framework/CurveFitting/src/Algorithms/Fit.cpp

```diff
@@ -48,6 +48,9 @@ void Fit::initConcrete() {
       ->setDocumentation("Whether the fit was successful");
   declareProperty("OutputChi2overDoF", 0.0, "Returns the goodness of the fit",
                   Kernel::Direction::Output);
+  declareProperty("OutputNIterations", 0, Kernel::Direction::Output);
```

Comment:
This breaks the Python interface and potentially user scripts. Can this be avoided somehow?

To me it looks like if **OutputNormalisedCovarianceMatrix** and **OutputParameters** properties were declare in `init()` and not in `createOutput()`, **OutputNIterations** could be added as the very last output property and keep the Python interface compatible. I might be overlooking some details here, though.